app/views/application/nav/account.erb

```diff
@@ -7,6 +7,13 @@
   <ul class="dropdown-menu" role="menu">
     <li><a role="menuitem" tabindex="-1" href="/<%= current_user.username %>">Profile</a></li>
     <li><a role="menuitem" tabindex="-1" href="/account">Account</a></li>
+    <% if current_user.teams && current_user.teams.any? %>
```

Comment:
The first check is redundant. You can just do `current_user.teams.any?` since `current_user.teams` is an ActiveRecord relation it won't ever be `nil`. You can also try `.exists?` since it will short circuit if it find at least one matching record (instead of loading all records).